support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierFirstStepLanding.tsx

```diff
@@ -0,0 +1,163 @@
+import { css, ThemeProvider } from '@emotion/react';
+import { cmp } from '@guardian/consent-management-platform';
+import { from, palette, space, until } from '@guardian/source-foundations';
+import {
+	Button,
+	buttonThemeReaderRevenueBrand,
+	Container,
+} from '@guardian/source-react-components';
+import {
+	FooterLinks,
+	FooterWithContents,
+} from '@guardian/source-react-components-development-kitchen';
+import { useEffect } from 'preact/hooks';
+import { useNavigate } from 'react-router';
+import CountryGroupSwitcher from 'components/countryGroupSwitcher/countryGroupSwitcher';
+import type { CountryGroupSwitcherProps } from 'components/countryGroupSwitcher/countryGroupSwitcher';
+import { CountrySwitcherContainer } from 'components/headers/simpleHeader/countrySwitcherContainer';
+import { Header } from 'components/headers/simpleHeader/simpleHeader';
+import { PageScaffold } from 'components/page/pageScaffold';
+import { useOtherAmountValidation } from 'helpers/customHooks/useFormValidation';
+import {
+	AUDCountries,
+	Canada,
+	EURCountries,
+	GBPCountries,
+	International,
+	NZDCountries,
+	UnitedStates,
+} from 'helpers/internationalisation/countryGroup';
+import { resetValidation } from 'helpers/redux/checkout/checkoutActions';
+import { getContributionType } from 'helpers/redux/checkout/product/selectors/productType';
+import { getUserSelectedAmount } from 'helpers/redux/checkout/product/selectors/selectedAmount';
+import {
+	useContributionsDispatch,
+	useContributionsSelector,
+} from 'helpers/redux/storeHooks';
+import { navigateWithPageView } from 'helpers/tracking/ophan';
+
+const checkoutBtnStyleOverrides = css`
+	width: 100%;
+	justify-content: center;
+`;
+
+const darkBackgroundContainerMobile = css`
+	display: flex;
+	background: linear-gradient(
```

Comment:
Could we move this into the `from.desktop` below and simply make this `background: ${palette.brand[400]};`? Think thats what the Figma background colour does.

Reply:
yeah I could do, although it will result in 1 extra line of css. There's an `until.tablet` query at the mo that sets the background colour to `palette.brand[400]` and also sets a border bottom. If I remove the `until.tablet`, make the background colour `palette.brand[400]` and have a default border bottom then in the new `from.tablet` query I would need to not only set the background linear gradient but also reset the border bottom

Reply:
Sounds like a good plan, want me to preview here or are you happy to do?